fhir/resource_structure_dev/encounter.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType" : "Encounter",
+  "id" : "encounter_example_hospital_admission_for_covid",
+  "meta" : {
+    "profile" : [
+      "our_url_here"
+    ]
+  },
+  "extension" : [
+    {
+      "url" : "our-research-study-url",
+      "valueReference": {"reference": "ResearchStudy/<example-research-study>"}
+    }
+  ],
+  "subject": {"reference": "Patient/<id>"},
+  "status": "completed",
+  "class" : [{
+    "coding" : [{
+      "system" : "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+      "code" : "IMP",
+      "display" : "inpatient encounter"
+    }]
+  }],
+  "actualPeriod": {"start": "2022-07-10", "end": "2022-07-22"},
+  "reason": [{
+    "value" : [{
+      "concept" : {
+        "coding" : [{
+          "system" : "http://snomed.info/sct",
+          "code" : "840539006",
+          "display" : "COVID-19"
+        }]
```

Comment:
partOf could be used to indicate an ICU admission is part of a more general hospital admission encounter